api/controllers/UserController.js

```diff
@@ -120,5 +120,22 @@ module.exports = _.mapValues({
     const params = req.allParams();
     Validation.requireParams(params, 'name');
     return res.ok(await Validation.usernameAvailable(params.name));
-  }
+  },
+  async forgotUsername (req, res) {
+    const params = req.allParams();
+    Validation.requireParams(params, 'email');
+    const users = await User.find({email: params.email});
+    res.send(202);
```

Comment:
I guess it's good you are being security conscious, but "worried about" is probably the wrong word there :-P